src/server/index.js

```diff
@@ -154,13 +154,15 @@ function responseCreateBusinessSuccess(res) {
 }
 
 function validateRequestBody(req) {
-  return (!req.body.name || !req.body.description ||
-    !req.body.imageUrl || !req.body.keyword
-    || !req.body.rating || !req.body.longitude
-    || !req.body.latitude);
+  return (!req.body.name || !req.body.description
+    || !req.body.address || !req.body.phone
+    || !req.body.keyword || !req.body.rating
+    || !req.body.longitude || !req.body.latitude
+    || !req.body.images);
 }
 
 app.post('/api/businesses', function(req, res) {
+  console.log("reqbody: " + req.body);
```

Comment:
add some more detail if we leave it here, otherwise we will have a lot of console.log that we don't now about
like:

`[POST /api/businesses] Request body: req.body`